src/modules/previewpane/MarkDownPreviewHandler/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("MarkdownPreviewHandler")]

```

Comment:
These would have to be changed to match the other PowerToys in the future whenever it is integrated into PowerToys. Just a heads up for something to keep track of.

Reply:
Sure!